app/assets/stylesheets/opportunities.scss

```diff
@@ -51,6 +51,12 @@ h1, h2 {
   }
 }
 
+.accordion-navigation a {
+  &:hover {
+    background-color: darken(#FF851B, 40%);
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`